lib/iruby/command.rb

```diff
@@ -99,7 +99,7 @@ def run_kernel
       @args.reject! {|arg| arg =~ /\A--log=(.*)\Z/ && IRuby.logger.loggers << Logger.new($1) }
       IRuby.logger.level = @args.delete('--debug') ? Logger::DEBUG : Logger::INFO
 
-      raise(ArgumentError, 'Not enough arguments to the kernel') if @args.size < 2 || @args.size > 4
+      raise(ArgumentError, 'Not enough arguments to the kernel') unless @args.size == 3
```

Comment:
Could you please change the message to `Wrong number of arguments to the kernel`

Reply:
Fixed. Thanks!